### PR TITLE
Feat: landing page and few refactor

### PR DESCRIPTION
**Pull request by krishanu2911**
1. Landing page Added
2. Changed a class name in the button css file
3. Added a single color in color css file
4. Image added in assets

https://user-images.githubusercontent.com/75560307/152617459-936f2d62-2c33-4903-b153-9f546a4be350.mp4

### CODE REVIEW

File: Doc-css/index.css [hunk 1]
```diff
@@ -0,0 +1,105 @@
+main{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
```
Comment (kjais1720) [inline Doc-css/index.css:4]: extract this into a utility class

File: Doc-css/index.css [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    background-color: var(--background-color);
+    width: 100vw;
+    min-height: 100vh;
+}
+.footer-div{
+    display: flex;
+    justify-content: center;
+    align-items: center;
```
Comment (kjais1720) [inline Doc-css/index.css:12]: same as above

File: Doc-css/index.css [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+}
+.navbar-icon:hover{
+    color: var(--paragraph-color);
+}
+.main-content-area{
+    margin-top: 2rem;
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
```
Comment (kjais1720) [inline Doc-css/index.css:49]: same as above

File: Doc-css/index.css [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+.text-div{
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+    width: 40rem;
+}
+.hero-img-div{
+    display: flex;
+    justify-content: center;
+    align-items: center;
```
Comment (kjais1720) [inline Doc-css/index.css:66]: same as above

**Review by shuntzhere — APPROVED** [review]
A small suggestion about standard screen sizes while making responsive screens.

File: index.html [hunk 5]
```diff
@@ -4,9 +4,43 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./CSS/base.css">
+    <link rel="stylesheet" href="./Doc-css/index.css">
     <title>Document</title>
```
Comment (coldpigli) [inline index.html]: Change title to name of your Library here.

**Review by coldpigli — COMMENTED** [review]
Suggested a minor change

File: index.html (hunk 5 above)
Comment (krishanu2911) [reply index.html]: thanks for pointing out just forget about that.
Changed 👍

**Pull request merged**